### PR TITLE
Implement hello world

### PR DESCRIPTION
**Pull request by DoctorRoot**
@DoctorRoot I've tried implementing the solution, but I can't get my code to compile. Can you take a look and let me know what I've done wrong?

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Hello
+
+  def say_hello
+    puts "Hello, world
```
Comment (emtompsett) [inline main.rb]: Need a close quote!

Comment (tdelaub) [inline main.rb]: You need an ending quote on Hello, world

Comment (robik123) [inline main.rb]: missing "

Comment (sigcsedemoaccount) [inline main.rb]: You left off the closing quote.

**Pull request merged**